terraform/stacks/main/outputs.tf

```diff
@@ -193,10 +193,10 @@ output "rds_subnet_az2" {
     value = "${module.stack.rds_subnet_az2}"
 }
 output "rds_subnet_cidr_az1" {
-  value = "${var.rds_private_cidr_1}"
+  value = "${module.stack.rds_private_cidr_1}"
```

Comment:
Looks like this isn't an output of the `stack` module--I think we'll need to add it.